Dockerfile

```diff
@@ -1,73 +1,27 @@
-FROM ruby:2.3-alpine
+FROM codeclimate/codeclimate-parser:java-beta
```

Comment:
if our channel release strategy is to make the tag based on the branch name, and not the build number, how do we bump the base image?

This looks correct based on our parser release strategy, but I'm just curious how we're planning to handle that...

Reply:
That's a good point. We probably shouldn't be using a `java-beta` tag for the codeclimate-parser image. I'll make some changes there, so we can use a `b###` tag here as well.

Reply:
Updated the `java` branch of `codeclimate-parser` to push `b###`-tagged images that we can use as the base for this channel: https://circleci.com/gh/codeclimate/codeclimate-parser/264